### ISSUE TITLE
[BUG] - Firefox extension warnings from store validations

### ISSUE THREAD

**Issue by piraces** [piraces/tldr-extension-browser#4]
**Describe the bug**
Several warnings reported by Firefox addons "store".

**To Reproduce**
1. Publishing the extension to the Firefox Add-ons center.

**Expected behavior**
No warnings and validation passing...

**Screenshots**

**Desktop (please complete the following information):**
 - OS: All
 - Browser: Firefox
 - Version: All

**Additional context**
There are several warnings when publishing the extension to the Firefox store and the latest version has been disabled due to a specific one:

```text
Unsafe assignment to innerHTML (Variable 'markdown' reassigned with unsafe value at 1:3571)
Warning: Due to both security and performance concerns, this may not be set using dynamic values which have not been adequately sanitized. This can lead to security issues or fairly serious performance degradation.

main.js line 1 column 3745
```

**All the warnings (6):**
```text
Manifest key not supported by the specified minimum Firefox version
Warning: "strict_min_version" requires Firefox 48.0, which was released before version 52 introduced support for "author".

manifest.json
Manifest key not supported by the specified minimum Firefox for Android version
Warning: "strict_min_version" requires Firefox for Android 48.0, which was released before version 52 introduced support for "author".

manifest.json
Manifest key not supported by the specified minimum Firefox for Android version
Warning: "strict_min_version" requires Firefox for Android 48.0, which was released before version 55 introduced support for "browser_action".

manifest.json
Manifest key not supported by the specified minimum Firefox for Android version
Warning: "strict_min_version" requires Firefox for Android 48.0, which was released before version 57 introduced support for "browser_action.default_popup".

manifest.json
Unsafe assignment to innerHTML (Variable 'markdown' reassigned with unsafe value at 1:3571)
Advertencia: Due to both security and performance concerns, this may not be set using dynamic values which have not been adequately sanitized. This can lead to security issues or fairly serious performance degradation.

main.js línea 1 columna 3745
Unsafe assignment to innerHTML
Warning: Due to both security and performance concerns, this may not be set using dynamic values which have not been adequately sanitized. This can lead to security issues or fairly serious performance degradation.

main.js línea 1 columna 4337
Permission not supported by the specified minimum Firefox version
Warning: "strict_min_version" requires Firefox 48.0, which was released before version 55 introduced support for "permissions:contextMenus".

manifest.json
```

**Comment by piraces** [comment]
Version 1.1.1 now in review for Edge, Chrome and Firefox markets.
Once published and verified this issue can be closed.

**Comment by piraces** [comment]
Closed as 1.1.1 is now available in all stores
